integration_tests/package.json

```diff
@@ -0,0 +1,9 @@
+{
+  "name": "integration_tests",
+  "description": "",
+  "version": "0.1.0",
+  "dependencies": {},
+  "devDependencies": {
+    "serverless-plugin-datadog": "^2.18.0"
```

Comment:
Is this reference to the plugin necessary?

Reply:
It is not, we are using the local version of the plugin anyways so I removed this dependency.